conductor-client/src/main/kotlin/com/openlattice/postgres/external/ExternalDatabasePermissioner.kt

```diff
@@ -369,13 +403,26 @@ class ExternalDatabasePermissioner(
 
             orgColumnAcls.forEach { columnAcl ->
                 val column = columnsById.getValue(columnAcl.aclKey)
-                val columnName = externalColumns.getValue(column.columnId)?.name ?: String()
-                val tableName = externalTables.getValue(column.tableId)?.name ?: String()
+                val columnName = if (securableObjectType == SecurableObjectType.OrganizationExternalDatabaseColumn) {
+                    externalColumns.getValue(column.tableId)?.name ?: String()
```

Comment:
Shouldn't this be `columnId`?

```suggestion
                    externalColumns.getValue(column.columnId)?.name ?: String()
```